zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/Configuration.java

```diff
@@ -37,6 +38,7 @@ public class Configuration {
     private static final String KAFKA_CONTENT_UPDATED_TOPIC = "content-updated";
     private static final String DATASET_API_AUTH_TOKEN = "FD0108EA-825D-411C-9B1D-41EF7727F465";
     private static final String SERVICE_AUTH_TOKEN = "15C0E4EE-777F-4C61-8CDB-2898CEB34657";
+    private static final String LEGACY_CACHE_API_SERVICE_AUTH_TOKEN = "748896205c3b42b43adb4b22fff11784c5d971187f280ab1b6f142c3d69e64e4";
```

Comment:
Out of curiosity, how come this token looks quite different to the other two above?

Reply:
I would have to ask ONS.  A Good question.  The long looks like a JWT.  Not sure one is 32 characters the long is 64 characters. Haven't had the time to look into this.

Reply:
Asking Linden he thinks those long tokens are part of the newer authentication they transforming to or using now. The shorter ones are from the old authentication.